scripts/configure-installer-tests-cluster.sh

```diff
@@ -75,10 +77,17 @@ spec:
         name: ${HTPASSWD_SECRET}
 EOF
 
-# TODO : Find better way to check application of settings on cluster
-sleep ${SLEEP_AFTER_SECRET_CREATION}
-
 # Login as developer and setup project
-oc login -u developer -p $USERPASS
+for i in {1..30}; do
+    oc login -u developer -p $USERPASS &> /dev/null
+    if [ $? -eq 0 ]; then
+        break
+    fi
+    sleep 2
+done
+
 oc new-project myproject
 sleep 4
+
+make bin
```

Comment:
Why you add make bin at the end of the script. Are you copying it to the prow setup after that through release script ?

Reply:
Will remove once make bin is added to tests
this is a TODO

Reply:
https://github.com/openshift/release/pull/3100